airbyte-integrations/connectors/source-monday/unit_tests/test_graphql_requester.py

```diff
@@ -150,3 +151,16 @@ def test_build_items_incremental_query(monday_requester):
     built_query = monday_requester._build_items_incremental_query(object_name, field_schema, stream_slice)
 
     assert built_query == 'items(limit:100,ids:[1, 2, 3]){id,name}'
+
+def test_get_request_headers(monday_requester):
+
+    object_name = "test_items"
```

Comment:
Why are `object_name`, `field_schema` and `stream_slice` defined? They don't seem to be used

Reply:
removed